wallet-new/src/Cardano/Wallet/API/V1/Handlers/Accounts.hs

```diff
@@ -0,0 +1,94 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TypeOperators     #-}
+
+module Cardano.Wallet.API.V1.Handlers.Accounts where
+
+import           Universum
+
+import           Servant
+
+import           Cardano.Wallet.WalletLayer.Types (PassiveWalletLayer (..))
+import qualified Cardano.Wallet.WalletLayer.Types as WalletLayer
+
+import           Cardano.Wallet.API.Request
+import           Cardano.Wallet.API.Response
+import qualified Cardano.Wallet.API.V1.Accounts as Accounts
+import           Cardano.Wallet.API.V1.Types
+import qualified Cardano.Wallet.Kernel.DB.Util.IxSet as KernelIxSet
+import qualified Data.IxSet.Typed as IxSet
+
+handlers :: PassiveWalletLayer IO -> ServerT Accounts.API Handler
+handlers w =  deleteAccount w
+         :<|> getAccount w
+         :<|> listAccounts w
+         :<|> newAccount w
+         :<|> updateAccount w
+         :<|> redeemAda w
+
+deleteAccount :: PassiveWalletLayer IO
+              -> WalletId
+              -> AccountIndex
+              -> Handler NoContent
+deleteAccount layer wId accIdx = do
+    res <- liftIO $ (WalletLayer.deleteAccount layer) wId accIdx
```

Comment:
Just out of curiosity, why the brackets?

Reply:
ah, I think it was just for visually separating out the function we get from the layer from the params, but no need for that, will amend 😉